coriolisclient/cli/providers.py

```diff
@@ -65,6 +71,15 @@ def _get_formatted_data(self, obj):
         return data
 
 
+class ProviderSchemasFormatter(formatter.EntityFormatter):
+    """ Formats dict-like entities with keys 'type' and 'schema'. """
+
+    columns = ("Schema Type", "Schema")
+
+    def _get_formatted_data(self, obj):
+        return (obj['type'], obj['schema'])
```

Comment:
Can you please run the command with `-f json/yaml` to ensure the structure is as we expect in case anyone'll be parsing/piping CLI output.

Reply:
The output: [https://pastebin.com/eYtrg319](url)